src/compiler/clang.rs

```diff
@@ -573,6 +573,37 @@ mod test {
         );
     }
 
+    #[test]
+    fn test_parse_xclang_plugin_arg_unknown_flag() {
```

Comment:
we have now actual integration tests with clang.
Could you please replicate this test with a plugin:
https://github.com/mozilla/sccache/blob/main/.github/workflows/integration-tests.yml#L403-L452
thanks

Reply:
There are no plugins available by default that I know of, using a plugin would require writing one and building it (or at least getting the binary for it).
Is it not enough that the plugin flags are in the `common_args` and that there are tests that verify these are part of the hash calculation?